src/frontend/src/views/Data/index.tsx

```diff
@@ -87,58 +107,59 @@ const Data: FunctionComponent = () => {
 
   const dataJSX: Record<string, Array<JSX.Element>> = {
     menuItems: [],
-    routes: [],
   };
 
   // create JSX elements from categories
   dataCategories.forEach((category) => {
     let focusStyle = null;
-    if (window.location.pathname === category.to) {
+
+    if (router.asPath === category.to) {
       focusStyle = style.active;
     }
+
     dataJSX.menuItems.push(
-      <Link to={category.to} key={category.text}>
-        <Menu.Item className={style.menuItem}>
-          <div className={style.category}>
-            <div className={cx(style.title, focusStyle)}>{category.text}</div>
-            <div className={style.count}>{category.data?.length}</div>
-          </div>
-        </Menu.Item>
+      <Link href={category.to} key={category.text} passHref>
```

Comment:
`next/link` needs `passHref` prop to pass its `href` attribute to its direct child anchor tag `<a />`, so we add `<a href="passHref"` as a convention that it will get the `href` from its Link parent